### ISSUE TITLE
disease gene research spotlight

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1112]

**Comment by ValWood** [comment]
yup I changed the title after I posted, but its 'explore pombase'

**Comment by kimrutherford** [comment]
That's live now.  The text is is pombe-embl/website/pombase_v2_config.json is you'd like to change anything.  Let me know if you edit it so I can re-release.
